### ISSUE TITLE
FMX, rotating 90 degrees a lottie animationcontrol, fails to play

### ISSUE THREAD

**Issue by gftecsi** [skia4delphi/skia4delphi#151]
I suppose that this is due to absoluteheight or width <0; this produces that the CanRunAnimation function returns false.
I patched this just changing this comparison to
(AbsoluteWidth <> 0) and (AbsoluteHeight <> 0)

**Comment by viniciusfbb** [comment]
Problem confirmed! The current comparison:

```delphi
    (AbsoluteWidth <> 0) and (AbsoluteHeight <> 0) and
```

needs to be replaced by:

```delphi
    (not SameValue(AbsoluteWidth, 0, TEpsilon.Position)) and
    (not SameValue(AbsoluteHeight, 0, TEpsilon.Position)) and
```

**Comment by viniciusfbb** [comment]
Fixed in [new release v4.0.0](https://github.com/skia4delphi/skia4delphi/releases)
